server/src/main/java/fr/uga/l3miage/tp2/exo1/models/DeveloperMiage.java

```diff
@@ -0,0 +1,22 @@
+package fr.uga.l3miage.tp2.exo1.models;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+@Table(name = "developer_miage")
+public class DeveloperMiage {
+
+    @Id
+    @Column(name = "lastname")
+    private String lastname;
+
+    @Column(name = "firstname")
+    private String firstname;
+
+    @Column(name = "email")
+    private String email;
+
+    @ManyToMany(mappedBy = "developerMiages")
+    private Set<Projet> projets;
```

Comment:
Ta relation est à l'envers, la tu une table `projects_developper_maige` alors que je voulais l'inverse

donc il faut mettre le `mappedBy` de l'autre coté

```suggestion
    @ManyToMany
    private Set<Projet> projets;
```

Reply:
Oh je ne le voyais pas comme ça, je me disais que c'était la même chose vu que c'est juste une classe d'association qui a les 2 clés. Néanmoins j'en prends compte.